tests/configfiles/dockerfile_lint_test_files/default_rule.yaml

```diff
@@ -95,7 +95,16 @@
             - "#from"
     MAINTAINER: 
       paramSyntaxRegex: /.+/
-      rules: []
+      rules:
```

Comment:
this is all junk.  https://github.com/coala/coala-bears/issues/2331

no benefit in 'fixing it'

Reply:
what am I supposed to do now :v ?

Reply:
i.e. this change is doing more of https://github.com/coala/coala-bears/pull/2314 , when we should be doing less.